project/MiMa.scala

```diff
@@ -33,15 +33,19 @@ object MiMa extends AutoPlugin {
         else if (projectName.contains("coordination")) 22
         else 0
 
-      if (!(projectName.contains("typed") || projectName.contains("jackson"))) {
-        // 2.5.18 is the only release built with Scala 2.12.7, which due to
-        // https://github.com/scala/bug/issues/11207 produced many more
-        // static methods than expected. These are hard to filter out, so
-        // we exclude it here and rely on the checks for 2.5.17 and 2.5.19.
-        expandVersions(2, 5, ((firstPatchOf25 to latestPatchOf25).toSet - 18).toList)
-      } else {
-        Nil
-      } ++ expandVersions(2, 6, 0 to latestPatchOf26)
```

Comment:
Oops, missing parentheses around all of the if statement.

Reply:
For fun, I had a look at how Dotty deal with this, without brackets at all:

```scala
scala> {
     |
     | if true
     |   List(1)
     | else
     |   List(2)
     | ++ List(3)
     |
     | }
val res5: List[Int] = List(1)

scala> {
     |
     | if false
     |   List(1)
     | else
     |   List(2)
     | ++ List(3)
     |
     | }
val res6: List[Int] = List(2, 3)
```

Reply:
Oops, that seems even more wrong that the current one.

Reply:
Ah, that's not even with the new bracket-less syntax, right? Because without brackets, it should change behavior again?

Reply:
Because then it would have to use indentation to imply the nesting.

Reply:
I wanted to test the bracket-less syntax, using brackets only to be able to write it in the REPL.  But I don't know if that affects things.  So I tried again with a simple file:

```scala
object Main

  def main(args: Array[String]): Unit =
    println(foo(false))
    println(foo(true))

  def foo(b: Boolean) =
    if b
      List(1)
    else
      List(2)
    ++ List(3)

end Main
```

same results:

```
15:07:24 $ bin/dotc Main.scala

15:07:32 $ scala Main
List(2, 3)
List(1)
```

So, yeah, the "append List(3) still only applies to the else expression only (incorrectly IMO).